packages/upstream-extra/ocveralls.0.3.4/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/sagotch/ocveralls/archive/3.3.4.tar.gz"
```

Comment:
Here for the original files: https://github.com/ocaml/opam-repository/tree/master/packages/ocveralls/ocveralls.0.3.4